### DISCUSSION TITLE
Demo Data

### DISCUSSION THREAD

**Discussion by EslamAsHhraf** [camicroscope/caMicroscope#670] [Q&A] [answered]
After Installation and set database. I want to add some demo data to use it, How I can get some data? 

![image](https://github.com/camicroscope/caMicroscope/assets/71986226/ed833c1d-e606-431a-b31a-c4bf4d03e859)

**Comment by birm** [comment]
I like this idea, not just for the workbench, but also for some example slides/annots/heatmaps to show all properties of caMicroscope!

**Reply by EslamAsHhraf** [reply]
@birm  I thought you are the man who can provide some demo data 😂

**Reply by birm** [reply, marked as answer]
I have the data I used to use to smoke test here (https://github.com/camicroscope/Distro/blob/master/config/test_seed.js) but it's not really like real data. We'd need to collect something we have rights/access to. It's a good idea but I don't have anything ready that covers all ouf our data types

**Reply by EslamAsHhraf** [reply]
Thanks Brim
